src/requirements/graph/processor/copy-graph.ts

```diff
@@ -0,0 +1,14 @@
+import GraphProcessor from './definition';
```

Comment:
nit - it's really satisfying how all of your files in the processor folder are formatted similarly. is there anywhere else you can put this? a utils folder by chance?

Reply:
hmm I think it actually looks nice in the folder